### ISSUE TITLE
Menu Edit>Search for property won't work

### ISSUE THREAD

**Issue by damiensellier** [damiensellier/CtrlrX#52]
Check why Menu Edit>Search for property is not working properly and if it used to do something in v5.3.198.

<img width="237" alt="Capture d’écran 2024-12-29 à 12 58 50" src="https://github.com/user-attachments/assets/c16a8b37-40f2-443e-966e-ec3e36564105" />

**Comment by damiensellier** [comment]
CtrlrEditorApplicationCommandsMenu.cpp L152

```
		// menu.addSeparator(); // Removed v5.6.31
		// menu.addCommandItem(commandManager, doSearchForProperty); // Removed v5.6.31
```

**Comment by damiensellier** [comment]
FIXED on 29.12.2024 for v5.6.31 (REMOVED)

CtrlrEditorApplicationCommandsMenu.cpp L152

![Capture d’écran, le 2024-12-29 à 14 03 33](https://github.com/user-attachments/assets/967ffc03-a917-44c9-bde6-a349bfae08c1)
